desktop/release-notes.md

```diff
@@ -24,12 +24,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.15.0
+
+{% include release-date.html date="2022-12-01" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Substantial performance improvements for macOS users with the option of enabling the new VirtioFS file sharing technology. Available for macOS 12.5 and above. 
+- Docker Desktop for Mac no longer needs to install the privileged helper process `com.docker.vmnetd` on install or on the first run. For more information see [Permission requirements for Mac](https://docs.docker.com/desktop/mac/permission-requirements/).
+- Added [WebAssembly capabilities](wasm/index.md). Use with the [containerd integration](containerd/index.md).
+- Improved the descriptions for beta and experimental settings to clearly explain the differences and how people can access them.
+- Available disk space of VM now displays in the footer of Docker Dashboard.
```

Comment:
This is only available for mac and linux. Should that info be added?

Reply:
If we look at the footer notes for 4.9.0

https://docs.docker.com/desktop/release-notes/#490

We didn't mentioned it was not fully working on some platforms (eg: it's not showing stats for Windows containers), so I guess it's fine keeping it like this.

@aevesdocker any recommendation maybe?

Reply:
I don't think it would hurt to add 'for Mac and Linux' at the end of the sentence

Reply:
Added!